### PR TITLE
Add warning for data services functions

### PR DESCRIPTION
**Pull request by elenatorro**
Related issue: https://github.com/CartoDB/support/issues/2046

Screenshot 2019-09-04 at 11 01 24

Screenshot 2019-09-04 at 11 01 10

### CODE REVIEW

File: lib/assets/javascripts/builder/components/code-mirror/data-services.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module.exports = [
+  'cdb_geocode_admin0_polygon',
```
Comment (jgoizueta) [inline lib/assets/javascripts/builder/components/code-mirror/data-services.js]: IIRC only street level geocoding consumes credits (not admin0, admin1, namedplace, postalcode, ipaddress, which uses our own internal service).

Comment (jgoizueta) [reply lib/assets/javascripts/builder/components/code-mirror/data-services.js]: So this should be: 
```ruby
module.exports = [
  'cdb_geocode_street_point',
  'cdb_isodistance',
  'cdb_isochrone',
  'OBS_GetDemographicSnapshot',
  'cdb_route_point_to_point',
  'cdb_route_with_waypoints'
];
```

File: lib/assets/test/spec/builder/components/code-mirror/code-mirror-view.spec.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ function fakeCodeMirrorKey (cm, type, code, props) {
   }
 }
 
-describe('components/code-mirror/code-mirror-view', function () {
+fdescribe('components/code-mirror/code-mirror-view', function () {
```
Comment (jgoizueta) [inline lib/assets/test/spec/builder/components/code-mirror/code-mirror-view.spec.js]: Hey, did you forget to remove that test _focus_ ?

Comment (alejandraarri) [reply lib/assets/test/spec/builder/components/code-mirror/code-mirror-view.spec.js]: Good catch! Removing it right now!

**Comment by alejandraarri** [comment]
Retest this please

**Pull request merged**